packages/nitro-protocol/src/transactions.ts

```diff
@@ -52,6 +53,17 @@ export function createCheckpointTransaction(signedStates: SignedState[]): Transa
   });
 }
 
+export function createConcludePushOutcomeAndTransferAllTransaction(
```

Comment:
We could add a test to `transactions.test` for this (although the test doesn't have a ton of value)